setup.py

```diff
@@ -121,6 +121,9 @@ class SpecialDependencies:
         "graphviz",
         "gridx-prescient>=2.2.1",
         "nrel-pysam>=3.0.1",
+        "tslearn>=0.5.2",
```

Comment:
The `dynamic_sweep` data package need to be installed for it to work in CI:

```suggestion
        "dispatches-dynamic-sweep-data @ git+https://github.com/gmlc-dispatches/dynamic-sweep-data@main",
        "tslearn>=0.5.2",
```